src/org/zalando/stups/mint/worker/job/run.clj

```diff
@@ -37,6 +41,7 @@
           (doseq [mint-app mint-apps]
             (let [app-id (:id mint-app)
                   kio-app (get kio-apps-by-id app-id)]
+              (when (or (not etcd-lock-url) (etcd/refresh-lock etcd-lock-url worker-id 600))
```

Comment:
I will add the configurable TTL, I will keep the `when` clause where it is (I cannot do it above the `doseq` as the lock might time out)